javascript_yhz/U5/hw5.js

```diff
@@ -0,0 +1,75 @@
+//1
+const x = 5;
+function squareRoot(num) {
+  console.log(Math.sqrt(num));
+  return Math.sqrt(num);
+}
+squareRoot(x);
+
+//2
+const arr1 = [
+  1, 0, 1, 1, 1, 0, 1, 1, 1, 1, 1, 1, 1, 0, 1, 0, 1, 1, 0, 0, 0, 0, 0, 0, 1,
+];
+function numOflongest1(arr) {
+  const only1 = arr
+    .join("")
+    .split(0)
+    .filter((item) => item !== "")
+    .sort((a, b) => b.length - a.length);
```

Comment:
you don't have to `sort` here, just use `Math.max` to get the maximum length of string